building/config.json

```
@@ -387,6 +387,19 @@
     "path": "building/tracks/stories/tuples.santas-helper.md",
     "title": "Santa's Helper"
   },
+  {
+    "uuid": "2eadb0af-4b2f-4598-a5b2-6b9cdf4a5c1b",
+    "slug": "tracks/ci",
+    "path": "building/tracks/ci/README.md",
+    "title": "Continuous Integration",
+    "blurb": ""
```

Why the empty blurb?

---

Wasn't sure what to put there. A lot of entries in the file don't have a blurb.

---

I think leave it empty for now? @iHiD

---

```suggestion
    "blurb": "Learn about how Exercism handles Continuous Integration through GitHub Actions"
```

(It's useful from a searching perspective to have these - both in terms of our own internal search while will weight blurb highly, but also from an external people Googling POV. I'm happy to write them if people ping me 🙂)